cluster/gce/gci/configure.sh

```diff
@@ -334,7 +335,14 @@ function install-crictl {
     local -r crictl_hash="${CRICTL_TAR_HASH}"
   else
     local -r crictl_version="${DEFAULT_CRICTL_VERSION}"
-    local -r crictl_hash="${DEFAULT_CRICTL_SHA512}"
+    case "$HOST_PLATFORM-$HOST_ARCH" in
+      linux-amd64)
+        local -r crictl_hash="${DEFAULT_CRICTL_AMD64_SHA512}"
+        ;;
+      linux-arm64)
+        local -r crictl_hash="${DEFAULT_CRICTL_ARM64_SHA512}"
+        ;;
+    esac
```

Comment:
should this set to non-empty for unknown platforms to fail in case it's used for something this script doesn't know about?

Reply:
Yep I can do that!

Should I add error handling if HOST_PLATFORM or HOST_ARCH are empty?

There's an "Assumptions" comment at the top of the function
```
# Assumptions: HOST_PLATFORM and HOST_ARCH are specified by calling detect_host_info.
```

Reply:
shouldn't be empty, no need to check.

I like this approach with exit 1 more than I did in NPD: https://github.com/kubernetes/kubernetes/blob/9288c190f952726287697f26fe146b98781fcf45/cluster/gce/gci/configure.sh#L275 Can you please update NPD arch check as well to be the same? Thank you!

Reply:
@SergeyKanzhelev , I have updated NPD to use a similar format. 

I noticed that the cni binary installer could support ARM and this architecture check format. I can update the CNI in another PR if that seems reasonable.